data/_ui/interfaces.txt

```diff
@@ -1292,6 +1293,81 @@ interface "bank"
 
 
 
+interface "shop"
+	anchor bottom right
+	value "fleet bar width" 250
+	value "item bar width" 300
+	box "leave button"
+		center -45 -25
+		dimensions 70 30
+	box "park button"
+		center -125 -25
+		dimensions 70 30
+	box "buy button"
+		center -505 -25
+		dimensions 70 30
+	box "sell button"
+		center -425 -25
+		dimensions 70 30
+	box "sell ship button"
+		center -215 -25
+		dimensions 50 30
+
+	button l "_Leave"
+		center -45 -25
+		dimensions 70 30
+		size 18
+
+	visible if "use buy"
+	active if "can buy"
+	button b "_Buy"
+		center -505 -25
+		dimensions 70 30
+		size 18
+
+	visible if "use install"
+	active if "can buy"
+	button b "_Install"
+		center -505 -25
+		dimensions 70 30
+		size 18
+
+	visible if "use cargo"
+	active if "can buy"
+	button b "_Cargo"
+		center -505 -25
+		dimensions 70 30
+		size 18
+
+	visible if "is outfitters"
```

Comment:
Shouldn't be singular?
```suggestion
	visible if "is outfitter"
```

Reply:
I would agree but the precedent has already been set and I didn't want to use two different values in two places in interfaces.txt as that makes maintenance and plugins harder.

Reply:
Precedent? Where?